test/js/api/notify-info.js

```diff
@@ -0,0 +1,60 @@
+const assert = require("assert");
+const uuid = require("uuid");
+const config = require("../config");
+const ripe = require("../../../src/js");
+
+describe("NotifyInfoAPI", function() {
+    this.timeout(config.TEST_TIMEOUT);
+
+    describe("#createDeviceId", function() {
+        beforeEach(function() {
+            if (!config.TEST_USERNAME || !config.TEST_PASSWORD) {
+                this.skip();
+            }
+        });
+
+        it("should be able to create a deviceID", async () => {

```

Comment:
Test also what happens when you create the same device ID twice. Unit tests are the time to get creative :)

Reply:
Also, try adding 2 different device IDs for the same user.